### PR TITLE
Edits to Slack, Before Starting, What you get

### PR DESCRIPTION
**Pull request by misseuropa**
Here's the next chunk. I rebased. Cherry pick what you're ok with.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,27 +21,23 @@ Table of contents
...
 
 # What you get from this control-repo
 
-As a result of following the instructions below you will receive the beginning of a best-practices installation of PE including...
+When you finish the instructions below, you will have the beginning of a best practices installation of PE including:
 
  - A Git server
  - The ability to push code to your Git server and have it automatically deployed to your PE master
- - A config_version script to output the commit of code that your agent just applied
+ - A config_version script that outputs the most recent sha and commit message for your code deployment each time you run `puppet agent -t`
```
Comment (npwalker) [inline README.md]: There's no commit message just the sha

File: README.md [hunk 2]
```diff
@@ -21,27 +21,23 @@ Table of contents
 
 # Join the #ramp-up channel on Puppet Community Slack
 
-Our [Puppet Community Slack](http://slack.puppet.com) is a great way to interact with other Puppet users.  The #ramp-up channel is specifically for users talking about starting with PE and using this repository.
-
-Other channels in the Puppet Community Slack are great for asking general Puppet questions as well.
+Our [Puppet Community Slack](http://slack.puppet.com) is a great way to interact with other Puppet users.  The #ramp-up channel is specifically for users who are using this repository who are new to Puppet Enterprise. Other channels in the Puppet Community Slack are great for asking general Puppet questions.
```
Comment (npwalker) [inline README.md]: looks like we're missing a word "who are using this repository who are new to Puppet Enterprise"

Comment (misseuropa) [inline README.md]: You're right, probably 'The #ramp-up channel is specifically for users who are new to Puppet Enterprise and using this repository' is better.

File: README.md (hunk 1 above)
Comment (misseuropa) [inline README.md]: Will take it out, wasn't quite sure. Will make the changes this morning

**Comment by misseuropa** [comment]
Updates should address your comments.

**Pull request merged**